docs/source/faq.md

```diff
@@ -115,6 +115,33 @@ It allows customization of the various parameters. An environment variable
 has precedence over the value defined in `ucx.conf`.
 The file can be created using `ucx_info -Cf`.
 
+#### Build user application with UCX
+
+UCX supports metainformation subsystem based on `pkg-config` tool. To add UCX support
+into application, output from `pkg-config` tool may be used in Makefile:
+```
+program: program.c
+        cc program.c $(pkg-config --cflags --libs ucx)
+```
+To build static application with UCX supporting just specific network transport
```

Comment:
When linking with static UCX libraries, need to list explicitly all required transport modules.
For example, in order to support only cma and knem transports, can use this method: